### ISSUE TITLE
2020-03-05 meeting notes

### ISSUE THREAD

**Issue by agrover** [mozilla-necko/meeting-notes#62]
**Round table:**

- AppCache
- Cache and files
- Bug [1615698](https://bugzilla.mozilla.org/show_bug.cgi?id=1615698) - Browser caching does not work with ranges that don't start with zero
- Bug [1612701](https://bugzilla.mozilla.org/show_bug.cgi?id=1612701) - WebSocket blocked by uBlock Origin still does a DNS request

**Next on triage duty:** Michal

**Comment by KershawChang** [comment]
Landed:
- [Bug 1613873](https://bugzilla.mozilla.org/show_bug.cgi?id=1613873) - Create a simple http channel which is only used for TRR
- [Bug 1615335](https://bugzilla.mozilla.org/show_bug.cgi?id=1615335) - TRR: Use SimpleHttpChannel off main thread
- [Bug 1584104](https://bugzilla.mozilla.org/show_bug.cgi?id=1584104) - We need to have a clone of TransportSecurityInfo on the parent process

Ongoing:
-  [Bug 1527384](https://bugzilla.mozilla.org/show_bug.cgi?id=1527384) - [socket process] Make SpeculativeConnect work
- [Bug 1537761](https://bugzilla.mozilla.org/show_bug.cgi?id=1537761) - Make AlternateServices work when socket process is enabled
- [Bug 1619582](https://bugzilla.mozilla.org/show_bug.cgi?id=1619582) - TRR: Rename SimpleHttpChannel

**Comment by vonasek** [comment]
- [landed] 1614619 Eviction algorithm should first evict entries of a content type that’s above a limit
- 1587353 Add tests for http3

**Comment by agrover** [comment]
Neqo:
 * Neqo 0.2.0 tagged and merged into m-c
 * Interop: Testing client congestion control may require new tests

DoH canary
* Options for querying network ASN via MLS-like webservice

next:
* Neqo bugs & H3 testing
* More quic-network-interop
* Address comments on qlog PR
* Doc: Plan for Neqo features in future Firefox versions

**Comment by valenting** [comment]
[landed]
Bug [1616345](https://bugzil.la/1616345) - Clear DNS on HTTP 421
Bug [1618130](https://bugzil.la/1618130) - Add API to override platform DNS results
[working on]
Bug [1617889](https://bugzil.la/1617889) - Fix CrossOriginOpenerPolicy inheritance from a cross origin iframe

**Comment by ddragana** [comment]
Neqo: 
- QPack refactor in review
Necko:
- Bug 1619813 - Handle http3 streams being blocked on max_stream_data
- Bug 1619793 - Make sure we do not AltSvc mapping for Http3 if Http3 is disabled
- Bug 1619764 - Make Http3Session easier to understand
